src/Nowruz/modules/userProfile/components/about/education/useEducation.tsx

```diff
@@ -62,10 +64,26 @@ export const useEducation = () => {
     dispatch(setIdentityType('users'));
   };
 
-  const handleOpenRequestCertificate = () => {
+  const handleOpenRequestCertificate = async (ed: AdditionalRes) => {
+    const res = await getOrganization((ed.meta as EducationMeta).school_id);
+    setOrg(res);
+    setEducation(ed);
+    setOpenCertificate(true);
     return;
   };
 
+  const handleSendRequestCertificate = async (id: string, message?: string, exact_info?: boolean) => {
+    try {
+      // Apply new api from BE
+      // await requestVerifyExperience(id, message, exact_info);
+      const updated = await otherProfileByUsername(user?.username || '');
+      dispatch(setIdentity(updated));
+      dispatch(setIdentityType('users'));
+    } catch (e) {
+      console.log('error in verifying experiece:', e);
```

Comment:
this is not important but logging is for developers and better to specify which section has errors so: `error in verifying education` is the better log.